### PR TITLE
Clean Old Otps artisan command

### PR DESCRIPTION
**Pull request by bkd27**
Added artisan command to clean old, expired, used otps.

### CODE REVIEW

**Comment by ichtrojan** [comment]
Hi @bkd27,

Thanks a lot for this PR, before merging can you add instructions on how to use this on the README file

**Comment by bkd27** [comment]
Thank you for the quick response, I have updated all changes. Let me know if anything left.

File: README.md [hunk 1]
```diff
@@ -155,6 +155,21 @@ $otp = Otp::generate('michael@okoh.co.uk', '282581');
 }
 ```
 
+### Clean used Otps Artisan command
+Artisan command to clean used otps from database. 
+```bash
+php artisan otp:clean
+```
+You can also add this artisan command to `app/Console/Kernal.php` to automatically clean on scheduled
```
Comment (ichtrojan) [inline README.md:163]: change `Kernal.php` to `Kernel.php`

File: src/Commands/CleanOtps.php [hunk 2]
```diff
@@ -43,10 +43,11 @@ public function handle()
 
             $this->info("Found {$otps} expired otps.");
             Otp::where('valid', 0)->delete();
-            $this->info("Removed successfully expired  tokens!");
+            $this->info("Removed successfully expired tokens!");
```
Comment (ichtrojan) [inline src/Commands/CleanOtps.php:46]: Message should be:

`"expired tokens deleted"`

File: README.md [hunk 3]
```diff
@@ -155,6 +155,21 @@ $otp = Otp::generate('michael@okoh.co.uk', '282581');
 }
 ```
 
+### Clean used Otps Artisan command
```
Comment (ichtrojan) [inline README.md:158]: change to

`### Delete expired tokens`

File: README.md [hunk 4]
```diff
@@ -155,6 +155,21 @@ $otp = Otp::generate('michael@okoh.co.uk', '282581');
...
+Artisan command to clean used otps from database.
```
Comment (ichtrojan) [inline README.md:159]: Reword to:

`You can delete expired tokens by running the following artisan command:`

Comment (ichtrojan) [reply README.md:159]: don't forget to leave a new line

**Review by ichtrojan — COMMENTED** [review]
I left a couple of comments, I think that's all.

**Pull request merged**